src/libstd/keyword_docs.rs

```diff
@@ -925,8 +925,42 @@ mod mut_keyword {}
 //
 /// Make an item visible to others.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+///`pub` makes any module, function, or data structure accessible from inside
```

Comment:
Please add a leading column of whitespace after the `///`, like in the other lines.